packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -551,6 +554,12 @@ const TinyMCE = (props: NoteBodyEditorProps, ref: any) => {
 				'h1', 'h2', 'h3', 'hr', 'blockquote', 'table', `joplinInsertDateTime${toolbarPluginButtons}`,
 			];
 
+			const contentStyle = `
+				.jop-tinymce p {
```

Comment:
To avoid using "margin: 0 !impotant;" I've used .job-tinymce class to make the selector stronger.
Also, I prefered to create a const do put the CSS just to not put CSS inside the js object. I think that the code is cleaner like this.